cmd/query/app/grpc_handler_test.go

```diff
@@ -219,7 +228,7 @@ func initializeTestServerGRPCWithOptions(t *testing.T, options ...testOption) *g
 	logger := zap.NewNop()
 	tracer := opentracing.NoopTracer{}
 
-	server, addr := newGRPCServer(t, q, tqs.metricsQueryService, logger, tracer)
+	server, addr := newGRPCServer(t, q, tqs.metricsQueryService, logger, tracer, &tenancy.TenancyConfig{})
```

Comment:
Shouldn't be `tenancyConfig` passed instead of creating a new one?

Reply:
Now fixed, but of course that dismissed your review....